### ISSUE TITLE
allow running precheck script

### ISSUE THREAD

**Issue by mtbc** [ome/omego#112]
Now that we routinely offer a `sql/psql/new-version/old-version-precheck.sql` perhaps omego could support doing a precheck.

**Comment by manics** [comment]
Is `*-precheck.sql` guaranteed to be read-only, now and in future?

**Comment by joshmoore** [comment]
Logically, I'd say yes, but we should document this. For me, this is a process of adding "API" to the `sql/` directory.

**Comment by mtbc** [comment]
Agreed. I plan to always end it with `ROLLBACK;`. At very worst we *may* consider recording in `dbpatch` that it was run.
